docs/wymeditor_development/making_releases.rst

```diff
@@ -111,24 +118,15 @@ different files:
 Throughout these instructions, when asked to modify the version string, it is
 meant that the version strings be modified in all of these files.
 
-Remove the ``dev`` build metadata from the version string.
-Also, remove the ``+`` because there is no more build metadata.
-
-While you're doing that, make sure that the version strings are otherwise correct.
-
-If instructions here were followed,
-They should be already the version that you mean to release.
-
-Update the changelog
---------------------
+Come up with a version string for this release.
+Adhere to `SemVer`_.
 
-#. If this release makes a transition from alpha to beta or from beta to stable,
-   consolidate alpha/beta change–log entries in to a unified section
-   for this release.
+Bump the version string.
+This means replacing ``next``
+in the above files
+with the new version string.
 
-#. If this is a major release, highlight important changes.
-
-#. Set the date for the release.
```

Comment:
This is moved to just before committing changes, so in case you start release process one day and carry on a different day, you get the right date in.